query-engine/connector-test-kit-rs/query-engine-tests/tests/new/ref_actions/on_delete/set_default.rs

```diff
@@ -344,7 +344,7 @@ mod one2many_opt {
     }
 
     /// Deleting the parent reconnects the child to the default.
-    #[connector_test(schema(optional_with_default))]
+    #[connector_test(schema(optional_with_default), exclude(Vitess("planetscale.js")))]
     async fn delete_parent(runner: Runner) -> TestResult<()> {
```

Comment:
This seems to be running and failing for MongoDB now.
https://github.com/prisma/prisma-engines/actions/runs/6957501582/job/18930524260?pr=4423#step:6:2478